samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -0,0 +1,152 @@
+This means that you can use the J-Link COM port on the PC side to connect with nRF9160 DK, and send or receive AT commands there.
+
+* Terminal serial configuration:
+
+    * Hardware flow control: disabled
+    * Baud rate: 115200
+    * Parity bit: no
+
+.. note::
+   * The default AT command terminator is Carrier Return and Line Feed, i.e. ``\r\n``.
+   * nRF91 logs are output to the same terminal port.
+
+External MCU configuration
+==========================
+   
+To work with external MCU (nRF52), you must set the configuration option :option:`CONFIG_SLM_CONNECT_UART_2`.
+The pin interconnection between nRF91 and nRF52 is presented in the following table:
+
+.. list-table::
+   :align: center
+   :header-rows: 1
+
+   * - nRF52 DK
+     - nRF91 DK
+   * - UART TX P0.6
+     - UART RX P0.11
+   * - UART RX P0.8
+     - UART TX P0.10
+   * - UART CTS P0.7
+     - UART RTS P0.12
+   * - UART RTS P0.5
+     - UART RTS P0.13
+   * - GPIO OUT P0.27
+     - GPIO IN P0.31
+
+UART instance in use:
+
+    * nRF52840 and nRF52832 (UART0)
+    * nRF9160 (UART2)
+
+UART configuration:
+
+    * Hardware flow control: enabled
+    * Baud rate: 115200
+    * Parity bit: no
+    * Operation mode: IRQ
+
+Note that the GPIO output level on nRF91 side should be 3 V.
+
+TCP/IP AT commands
+******************
+
+The following proprietary TCP/IP AT commands are used in this sample:
+
+* AT#XSOCKET=<op>,<type>
+* AT%XSOCKET?
+* AT#XTCPCONN=<url>,<port>
+* AT#XTCPCONN?
+* AT#XTCPSEND=<data>
+* AT#XTCPRECV=<length>,<time>
+* AT#XUDPSENDTO=<url>,<port>,<data>
+* AT#XUDPRECVFROM=<url>,<port>,<length>,<time>
+
+GPS AT Commands
+***************
+
+The following proprietary GPS AT commands are used in this sample:
+
+
+* AT#XGPSRUN=<op>[,<mask>]
+* AT#XGPSRUN?
+
+Building and Running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/serial_lte_modem`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+The following configuration files are located in the :file:`samples/nrf9160/serial_lte_modem` directory:
+
+- :file:`prj.conf`
+  This is the standard default configuration file.
+- :file:`child_secure_partition_manager.conf`
+  This is the project-specific Secure Partition Manager configuration file.
+
+To test the sample with a PC client, open a terminal window and start sending AT commands to the nRF9160 DK.
+
+When testing the sample with an nRF52 client, the DKs go through the following start-up sequence:
+
+    1. nRF91 starts up and enters sleep state.
+    #. nRF52 starts up and starts a periodical timer to toggle the GPIO interface.
+    #. nRF52 deasserts the GPIO interface.
+    #. nRF91 is woken up and sends a ``Ready\r\n`` message to the nRF52.
+    #. On receiving the message, nRF52 can proceed to issue AT commands.
+
+You can find a sample nRF52 client project in the :file:`client` folder, which can be put under "SDK_15.3.0/examples/peripheral" folder.
```

Comment:
I can't understand why we are suddenly talking about nRF5 SDK (which doesn't support nRF91) in the context of this NCS sample?

Reply:
Sorry this is a left-over of old code, where a nRF52 project is kept inside this nRF91 project. It's removed based on review comment. I changed the line like this "A sample nRF52 client project is to be shared by other means."